Documentation/BuildInstructionsMacOS.md

```diff
@@ -36,6 +36,10 @@ brew install llvm@18
 brew install gcc@13
 ```
 
+# Nix
+
+If you have nix installed you can just enter the devShell provided by the flake and execute the run script.
```

Comment:
I think Nix is generally capitalized as "Nix" if it is not about the command itself.